lib/ServerDate.js

```diff
@@ -43,7 +43,37 @@ var
   precision,
   offset,
   target = null,
-  synchronizing = false;
+  synchronizing = false,
+
+  // ServerDate default configuration
+  config = {
```

Comment:
Do you want to remove the configuration options?. The only change will be the HEAD request instead of GET